### PR TITLE
fix(onboarding): Allow enter button to dismiss keyboard

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

I was setting up my new iPhone SE test device and could not even proceed past entering my name because the "Enter" key would not dismiss the keyboard and the keyboard was hiding the "Next" button. 🤯

IMG_0001

### Tested

Manually.

### How others should test

Start a new account, enter in your name, verify that the "Enter" button on the keyboard dismisses the keyboard.

### CODE REVIEW

File: packages/mobile/src/onboarding/registration/NameAndPicture.test.tsx [hunk 1]
```diff
@@ -14,26 +14,6 @@ expect.extend({ toBeDisabled })
...
-  it('renders with an error', () => {
-    const store = createMockStore({ alert: { underlyingError: ErrorMessages.INVALID_INVITATION } })
-    const tree = render(
-      <Provider store={store}>
-        <NameAndPicture {...mockScreenProps} />
-      </Provider>
-    )
-    expect(tree).toMatchSnapshot()
-  })
-
```
Comment (jh2oman) [inline packages/mobile/src/onboarding/registration/NameAndPicture.test.tsx:36]: These tests didn't really add any value. The snapshots for each are the same and subsequent tests verify the actual functionality

File: packages/mobile/src/onboarding/registration/NameAndPicture.tsx [hunk 2]
```diff
@@ -121,7 +121,7 @@ function NameAndPicture({ navigation }: Props) {
           enablesReturnKeyAutomatically={true}
           placeholder={t('fullNamePlaceholder')}
           testID={'NameEntry'}
-          multiline={true}
+          multiline={false}
```
Comment (jeanregisser) [inline packages/mobile/src/onboarding/registration/NameAndPicture.tsx:124]: oh interesting, this seems was inadvertently changed during accessibility fixes.

**Review by jeanregisser — APPROVED** [review]
Thanks! 👍

**Review by lisaqyli — APPROVED** [review]
oh wow that's nuts. Imagine the onboarding success rate going up haha

File: packages/mobile/src/onboarding/registration/NameAndPicture.tsx (hunk 2 above)
Comment (lisaqyli) [reply packages/mobile/src/onboarding/registration/NameAndPicture.tsx:124]: oh I'm not sure if this textalign top styling does anything here, is there any edge cases that needs to be tested? Maybe when the name input is really long? 
https://github.com/valora-inc/wallet/blob/029b7f6bdda17fabb4a7aae9b349f6c1e9730e38/packages/mobile/src/components/TextInput.tsx#L79

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Test Flight Release build V1.31.0 and updated status in test rails.

- User is able to dismissed the keyboard while tapping on enter key from the keyboard.

**Test rail links:**
**iOS15:** https://valoraapp.testrail.io/index.php?/tests/view/30609

**Devices:** iPhone 13 mini(15.1.1)

**CC:** @jh2oman 

Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT , We have verified above task on Latest Android internal release build and updated status in test rails.

Observation : User is able to dismissed the keyboard while tapping on enter key from the keyboard.

Test rail links:

Android 11: https://valoraapp.testrail.io/index.php?/tests/view/30702

Devices: Google Pixel 2XL(11.0)

CC: @jh2oman 
Thanks..!!

**Pull request merged**